styles/designs/webview/parts/_exercise-components.scss

```diff
@@ -111,6 +111,46 @@ $Exercise__Problem__MCAnswer__Pre: (
   )
 );
 
+
+$InjectedExercise__Question__MultiChoiceAnswers: (
+  _name: "InjectedQuestionAnswers",
+  _subselector: " ol[data-type=question-answers][type=a]",
+  _properties: (
+    counter-reset: multi-choice-index,
+    padding: 0,
+    margin: 0
+  )
+);
+
+$InjectedExercise__Question__MultiChoiceAnswers--Answer: (
+  _name: "InjectedQuestionAnswersAnswer",
+  _subselector: " > li[data-type=question-answer]",
+  _properties: (
+    display: table,
+    counter-increment: multi-choice-index,
+  )
+);
+
+$InjectedExercise__Question__MultiChoiceAnswers--AnswerLetter: (
+  _name: "InjectedQuestionAnswersAnswerLetter",
+  _subselector: " > li[data-type=question-answer]::before",
+  _properties: (
+    font-weight: bold,
+    color: enum('ValueSet:::REQUIRED'),
```

Comment:
Ah so is this how https://github.com/openstax/ce-styles/pull/562/files#diff-2ef9b46ee73f2b5f90a0c9af10b57b997895796e60818518792027746e463d8fR346 gets used?

Reply:
Yes, you can override properties elsewhere if they are `required` or `optional`. If they are set here like the font-weight, however, they cannot be changed.